recipe/meta.yaml

```
@@ -1,42 +1,48 @@
 {% set label = "'full'" %}
 {% set tests = "['pywt']" %}
-{% set name = "PyWavelets" %}
-{% set version = "1.4.1" %}
+{% set name = "pywavelets" %}
+{% set version = "1.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6437af3ddf083118c26d8f97ab43b0724b956c9f958e9ea788659f6a2834ba93
+  sha256: ea027c70977122c5fc27b2510f0a0d9528f9c3df6ea3e4c577ca55fd00325a5b
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Basing this loosely on what was [done for `scikit-image`]( https://github.com/conda-forge/scikit-image-feedstock/pull/101/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bc ). Though adjusted so that we hopefully keep everything in `script`

```suggestion
  script:
    - mkdir builddir
    - {{ PYTHON }} -m mesonbuild.mesonmain setup builddir
    - {{ PYTHON }} -m build --wheel --no-isolation --skip-dependency-check -Cbuilddir=builddir
    - pushd builddir
    - {{ PYTHON }} -m pip install *.whl
    - popd
```